libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/Navigation/Navigation.tsx

```diff
@@ -102,18 +108,42 @@ export class Navigation extends React.Component<INavigationProps> {
       }
     }
     const classNames = navigationStyles();
+    const messageBarStyles: IMessageBarStyles = {
+      content: classNames.content,
+      dismissal: classNames.dismissal,
+      dismissSingleLine: classNames.dismissSingleLine,
+      iconContainer: classNames.iconContainer,
+      text: classNames.text
+    };
     return (
       <div className={classNames.navigation}>
-        <div className={classNames.breadcrumb}>
-          <Breadcrumb
-            items={items}
-            maxDisplayedItems={10}
-            ariaLabel="Navigation"
-            overflowAriaLabel="More links"
-            onRenderItem={this._onRenderItem}
-            styles={breadcrumbStyle}
-          />
-        </div>
+        <Stack horizontal horizontalAlign="space-between">
+          <Stack.Item>
+            <div className={classNames.breadcrumb}>
```

Comment:
I removed the div but I also added another breadcrumbItemWidth to specify width of breadcrumb, otherwise it gets minized with the space-between horizontalAlign property.